virttest/utils_test/__init__.py

```diff
@@ -1849,8 +1866,11 @@ def __init__(self, vm, stress_type, download_url="", make_cmds="",
         :param download_type: currently support "git" or "tarball"
         """
 
-        self.vm = vm
-        self.params = vm.params
+        self.vm = None
+        self.session = None
+        self.remote_host = None
+        self.copy_files_to = shutil.copytree
```

Comment:
this will fail for default setting for host stress, we can `skip copy_files_to` method  incase of local host stress  and assign self.dst_path= source path ```
source: /var/tmp/avocado_nM3boR/stress-1.0.4  destination: /home
```
2018-06-05 06:06:55,300 stacktrace       L0041 ERROR|
2018-06-05 06:06:55,301 stacktrace       L0044 ERROR| Reproduced traceback from: /usr/lib/python2.7/site-packages/avocado_plugins_vt-61.0-py2.7.egg/avocado_vt/test.py:434
2018-06-05 06:06:55,303 stacktrace       L0047 ERROR| Traceback (most recent call last):
2018-06-05 06:06:55,303 stacktrace       L0047 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_plugins_vt-61.0-py2.7.egg/virttest/error_context.py", line 135, in new_fn
2018-06-05 06:06:55,303 stacktrace       L0047 ERROR|     return fn(*args, **kwargs)
2018-06-05 06:06:55,303 stacktrace       L0047 ERROR|   File "/var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-libvirt/libvirt/tests/src/multivm_stress/multivm_stress.py", line 17, in run
2018-06-05 06:06:55,303 stacktrace       L0047 ERROR|     utils_test.load_stress("stress_on_host", params=params)
2018-06-05 06:06:55,303 stacktrace       L0047 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_plugins_vt-61.0-py2.7.egg/virttest/utils_test/__init__.py", line 2141, in load_stress
2018-06-05 06:06:55,304 stacktrace       L0047 ERROR|     hstress.load_stress_tool()
2018-06-05 06:06:55,304 stacktrace       L0047 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_plugins_vt-61.0-py2.7.egg/virttest/utils_test/__init__.py", line 1835, in manage_session
2018-06-05 06:06:55,304 stacktrace       L0047 ERROR|     return func(self)
2018-06-05 06:06:55,304 stacktrace       L0047 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_plugins_vt-61.0-py2.7.egg/virttest/utils_test/__init__.py", line 1900, in load_stress_tool
2018-06-05 06:06:55,304 stacktrace       L0047 ERROR|     self.install()
2018-06-05 06:06:55,304 stacktrace       L0047 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_plugins_vt-61.0-py2.7.egg/virttest/utils_test/__init__.py", line 1989, in install
2018-06-05 06:06:55,304 stacktrace       L0047 ERROR|     self.download_stress()
2018-06-05 06:06:55,305 stacktrace       L0047 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_plugins_vt-61.0-py2.7.egg/virttest/utils_test/__init__.py", line 1983, in download_stress
2018-06-05 06:06:55,305 stacktrace       L0047 ERROR|     self.copy_files_to(source, self.dst_path)
2018-06-05 06:06:55,305 stacktrace       L0047 ERROR|   File "/usr/lib64/python2.7/shutil.py", line 191, in copytree
2018-06-05 06:06:55,305 stacktrace       L0047 ERROR|     os.makedirs(dst)
2018-06-05 06:06:55,305 stacktrace       L0047 ERROR|   File "/usr/lib64/python2.7/os.py", line 157, in makedirs
2018-06-05 06:06:55,305 stacktrace       L0047 ERROR|     mkdir(name, mode)
2018-06-05 06:06:55,306 stacktrace       L0047 ERROR| OSError: [Errno 17] File exists: '/home'
```